### PR TITLE
🔧 hacked translation list

### PR DESCRIPTION
**Pull request by Gerappa92**
It's not so trivial as I was thinking. Markdown sets argument `dir="auto"` to `ul` tag. 

According to the official [documentation ](https://www.w3.org/International/questions/qa-html-dir#dirauto)

> If the value of the dir attribute is set to auto, the browser will look at the first strongly typed character in the element and work out from that what the base direction of the element should be. If it's a Hebrew (or Arabic, etc.) character, the element will get a direction of rtl. If it's, say, a Latin character, the direction will be ltr.

Here is a related issue https://github.com/dear-github/dear-github/issues/147.

I decided to put this one item list into a separated `ul` HTML tag where I set `dir="ltr"`
Unfortunately, it makes an additional space below this item. Let me know if it is acceptable. I was trying to fix it using styles however, styles are not allowed in the Markdown.

https://stackoverflow.com/questions/20598628/do-style-tags-work-in-markdown

### CODE REVIEW

File: README-ca.md [hunk 1]
```diff
@@ -14,7 +14,10 @@
 
 Llista d'idiomes disponibles:
 
-- [العربية](./README-ar.md)
+<ul dir="ltr">
+    <li><a href="./README-ar.md">العربية</a></li>
+</ul>
+
```
Comment (nlepage) [inline README-ca.md:20]:
```suggestion
- [&lrm;العربية](./README-ar.md)
```

**Review by nlepage — CHANGES_REQUESTED** [review]
After a few research I found out about [left to right/right to left marks](https://www.w3.org/TR/WCAG20-TECHS/H34.html) which should do the trick.
I put an example suggestion in README-ca.md.

File: README-ca.md (hunk 1 above)
Comment (Gerappa92) [reply README-ca.md]: I was looking for something like that! 
Updated :)

**Review by nlepage — APPROVED** [review]
LGTM
fixes #112

**Pull request merged**